### PR TITLE
Create EIP155-282 for Cronos zkEVM Testnet

### PR DESCRIPTION
**Pull request by aw126**
Hello @ligi, this PR is to add the details of Cronos zkEVM Testnet on Chainlist (in addition to the current Cronos Chain). 
The previous PR https://github.com/ethereum-lists/chains/pull/4267 was closed due to inactivity.

Official document: https://docs.cronos.org/cronos-zkevm-chain/introduction-to-cronos-zkevm-chain-testnet

Network info:
```
Name: Cronos zkEVM Testnet
New RPC URL: https://rpc-zkevm-t0.cronos.org
Chain ID: 282
Symbol: TCRO
Block explorer URL: https://zkevm-t0.cronos.org/explorer
```

### CODE REVIEW

**Comment by aw126** [comment]
> I do not look at PRs with failing checks - and no I cannot remove this check as this would be a huge problem for EIP-3770

@ligi Thanks for the reply. The Build check does not pass with `short name must be unique - but found TCRO more than once`. It appears the failed check due to that it requires the symbol to be unique, but detected "TCRO" already exists for [Cronos (EVM) Testnet](https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-338.json) as symbol, so it does not allow the same symbol of TCRO exists for this new Cronos zkEVM Testnet.

Regarding this case of two different chains with the same symbol, would changing the `symbol` of one chain be the only solution? Kindly share if you have other suggestions. Thank you.

**Comment by ligi** [comment]
the *short name* MUST be unique

File: _data/chains/eip155-282.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Cronos zkEVM Testnet",
+  "chain": "CRO",
```
Comment (thomas-nguy) [inline _data/chains/eip155-282.json:3]:
```suggestion
  "chain": "CronosZkEVMTestnet",
```

File: _data/chains/eip155-282.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "rpc": ["https://rpc-zkevm-t0.cronos.org"],
+  "faucets": ["https://zkevm-t0.cronos.org/faucet"],
+  "nativeCurrency": {
+    "name": "Cronos zkEVM Test Coin",
+    "symbol": "TCRO",
+    "decimals": 18
+  },
+  "infoURL": "https://docs.cronos.org/cronos-zkevm-chain/introduction-to-cronos-zkevm-chain-testnet",
+  "shortName": "tcro",
```
Comment (thomas-nguy) [inline _data/chains/eip155-282.json:12]:
```suggestion
  "shortName": "zktcro",
```

**Comment by aw126** [comment]
Hi @ligi It should be good to go now. Thanks @thomas-nguy.

**Pull request merged**